lib/components/Attachments/index.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useRef, useEffect, useState } from "react";
+
+import classnames from "classnames";
+import { Button, Typography, Toastr } from "neetoui";
+
+import { DIRECT_UPLOAD_ENDPOINT } from "common/constants";
+import useUppyUploader from "hooks/useUppyUploader";
+
+import AttachmentCard from "./AttachmentCard";
+import AttachmentProgressCard from "./AttachmentProgressCard";
+import { DEFAULT_UPPY_CONFIG, UPPY_UPLOAD_CONFIG } from "./constants";
+
+const Attachment = ({
+  endpoint = DIRECT_UPLOAD_ENDPOINT,
+  attachments = [],
+  className,
+  onChange,
```

Comment:
```suggestion
  onChange: _ => {},
```
Giving a default value is necessary. We are explicitly calling the `onChange` method. It would throw an error if the host application does not provide this prop.